page_templates/style_guide.md

```diff
@@ -0,0 +1,73 @@
+
+# Jina AI Documentation Style Guide
+
+Guides are publicly available and represent the Jina AI brand and the quality of the product. Please adhere to the following steps to ensure basic professionalism in the docs.
+
+## Guiding principles:
+
+-   **Accessibility**. Everyone who needs to should understand our documents.
+
+-   **Professional**. A consistent writing and formatting style will give a professional look to the documents.
+
+## Specific guidelines:
+
+**[ For anything not covered here, we default to the [Google developer style guide.](https://developers.google.com/style)  ]**
+
+1.  Check the spelling and grammar in your contributions. Most editors include a spell checker or have an available spell-checking plugin. You can also paste your text into a Google Doc or other document software for a more robust spelling and grammar check. We use American English spelling.
+
+2.  Avoid run-on sentences. Generally, keep sentences to less than 13 words long.
```

Comment:
Can you link to a URL on run-on sentences? Some folks may not know what they are

Reply:
Good point, done